<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RaspberryPi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Logo raspberry pi.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Grafica_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GRAFICA 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Grafica_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GRAFICA 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Grafica_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GRAFICA 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Grafica_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GRAFICA 4</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SALIR">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 177, 181);</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATA GENERADA POR SENSORES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Grafica_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GRAFICA 5</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SALIR</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
